src/crypto-api/index.ts

```diff
@@ -516,6 +516,14 @@ export interface CryptoApi {
      */
     isKeyBackupTrusted(info: KeyBackupInfo): Promise<BackupTrustInfo>;
 
+    /**
+     * Get information about the current key backup.
```

Comment:
I copied the documentation of https://github.com/matrix-org/matrix-js-sdk/blob/develop/src/rust-crypto/backup.ts#L111 since we are just calling this function.